build/style-loaders.js

```diff
@@ -21,11 +21,14 @@ module.exports = [
 		loader: 'sass-loader',
 		options: {
 			implementation: sass,
-			includePaths: [
-				'src/assets/fonts',
-				'src/assets/scss',
-				'node_modules/foundation-sites/scss'
-			]
+			sassOptions: {
+				// fiber: false, // to disable automatic use of fibers package
```

Comment:
sass-loader 8 will use fibers by default if it's present. We can disable it if it's too finicky. I've found that the trick is to make sure you run npm i from within your vm to be sure the fibers post-install tasks run. After this it works fine and saves me 3-5 seconds on subsequent compilations...